duckduckgo/duckduckgo-privacy-extension
# Blue dax for beta builds

**Reviewer:**



## Description:
Blue icons for the beta channel:
<img width="421" alt="Screenshot 2022-11-30 at 09 03 35" src="https://user-images.githubusercontent.com/248111/204740713-51b45210-0417-4b58-9483-70be1844e9f9.png">
<img width="200" alt="Screenshot 2022-11-30 at 09 03 47" src="https://user-images.githubusercontent.com/248111/204740737-f085468f-ca55-4f99-91cb-19950cc9e6fc.png">

https://app.asana.com/0/892838074342800/1203446250622462/f

## Steps to test this PR:
1. `npm run beta-chrome-mv3`
2. Install the generated .zip file in Chrome.
3. Icons should be blue.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
